Nodejs/Product/TestAdapter/TestFrameworks/ExportRunner/exportrunner.js

```diff
@@ -85,6 +85,7 @@ var run_tests = function (testCases, callback) {
             result.title = test.testName;
             testCase[test.testName]();
             result.passed = true;
+            result.stdOut += "Test passed.\n";
```

Comment:
Yes it will appear on the output link. The catch clause already contains a console.error that will also write to the output link. We could change it to instead append to the result.stdErr, but it will be no different.